test/lib/study.js

```diff
@@ -55,6 +55,110 @@ module.exports = function(client, config) {
             .waitForPaginationComplete();
     };
 
+    me.view.visits = {
+        /**
+         * Fill out a study's visit form.
+         *
+         * This method works for both 'create' and 'edit' visit forms. The only
+         * thing that changes is the `form`'s `id`.
+         *
+         * @param  {Object} options Contains `data` and `mode` properties.
+         *   `data` contains input data for the form. `mode` holds the input
+         *   type of the form, either `add` for adding a new visit or `update`
+         *   for updating a visit. Example:
+         *
+         *       {
+         *           data: {
+         *               label: 'Test Label 2',
+         *               timeFromBaseline: 2,
+         *               timeUnit: 'Month'
+         *           },
+         *           mode: 'update'
+         *       }
+         *
+         * @return {Object}         Instance of `client`
+         */
+        fillOutForm: function (options) {
+            if (! options.data) {
```

Comment:
`!_options.data` ==> `!options.data`.

FYI, I still need to integrate our jscs into a pre-commit hook, using https://github.com/nlf/precommit-hook .  we have our standard here https://github.com/MRN-Code/javascriptStyleguide, i just haven't pieced it all together.  once we get that dialed, integrate it into https://github.com/MRN-Code/node_boilerplate

If you want to help out with that at some point, you're welcome to